x-pack/test/functional/page_objects/reporting_page.ts

```diff
@@ -164,6 +166,12 @@ export function ReportingPageProvider({ getService, getPageObjects }: FtrProvide
       const toTime = 'Sep 23, 1999 @ 18:31:44.000';
       await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
     }
+
+    async verifyDownloadCSVButton(panelSelector: string) {
+      const savedSearchPanel = await testSubjects.find(panelSelector);
+      await dashboardPanelActions.toggleContextMenu(savedSearchPanel);
+      await testSubjects.existOrFail('embeddablePanelAction-downloadCsvReport');
```

Comment:
I need some suggestions here on how to read the POST request parameters (URL and request body). It seems like the best place to do that is here, but I'm not sure how to access that data: The download button is populated in the context menu via a registry.

Reply:
Hey Tim. With #65476 in place it should be easy to simply download csv and check if it is in path:

```
const csvPath = resolve(__dirname, '../../../target/functional-tests/downloads/Ecommerce Data.csv')


await testSubjects.click('embeddablePanelAction-downloadCsvReport');
// waiting for Kibana toast, guess some delay for file to be saved as well
expect(fs.existsSync(path)).toBe(true);
```

Reply:
This is great! You are my hero 🦸‍♂️